src/commons_codec/transform/dynamodb.py

```diff
@@ -173,8 +173,24 @@ def values_to_update(self, keys: t.Dict[str, t.Dict[str, str]]) -> str:
             elif isinstance(key_value, str):
                 key_value = "'" + str(key_value).replace("'", "''") + "'"
 
-            if isinstance(key_value, dict):
-                key_value = repr(json.dumps(key_value))
+            elif isinstance(key_value, dict):
+                # TODO: Does it also need escaping of inner TEXT values, like the above?
```

Comment:
Single quotes must always be escaped as they are used for any text literals at CrateDB.

Reply:
Thanks. So that serialization will need further improvements to treat OBJECT values the same like above.